.github/workflows/openvex.yaml

```diff
@@ -0,0 +1,17 @@
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  vexctl:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - uses: openvex/generate-vex@31b415924ea0d72ed5f2640f1dee59dea6c2770b
+        name: Run vexctl
+        with:
+            product: pkg:generic/hello@${{ env.RELEASE_VERSION }}
```

Comment:
From what I can understand, this just generates the file, but doesn't actually attach it to the release. I think you need a separate upload step to attach the generated vex file to the release. See an example of that here: https://github.com/kubernetes-sigs/kueue/pull/1475

Reply:
@jonathan-innis: I've referred the Kueue PR and made the workflow manual trigger only. I've added the upload step as well. I was not able to trigger the workflow manually in my fork to test this though. I couldn't any logs of this being done in the Kueue repository as well. Please let me know if the code changes are fine.